org.lflang.tests/src/org/lflang/tests/runtime/TestBase.java

```diff
@@ -380,11 +380,13 @@ private boolean generateCode(LFTest test) {
 
             restoreOutputs();
 
-            // check if any errors occurred during code generation
-            if (DefaultErrorReporter.DEFAULT.getErrorsOccurred()) {
-                test.result = Result.CODE_GEN_FAIL;
-                return false;
-            }
+            // FIXME: This is incompatible with Clement's recent change 
+            // that removed DefaultErrorReporter.DEFAULT
+            // // check if any errors occurred during code generation
```

Comment:
In recent changes from @oowekyala that were merged into master, `DefaultErrorReporter.DEFAULT` was eliminated, so we need to create another way of keeping track of the error reporter instance, if we want this check here.

Reply:
I've fixed this, hopefully